server/config.js

```diff
@@ -14,7 +14,8 @@ const readConfig = (searchPath) => {
 }
 
 /** @type {{database: {enabled: Boolean, protocol: String, host: String, port: Number, username: String, password},games: Object.<String, {path: String, settings: {maxPlayers: Number, minPlayers: Number, timeout: Number, failClause: String, bot: String}, tournament: {}}>}} */
-let config = readConfig('config.json5')
+let path = process.env.NODE_ENV == "test" ? "config.test.json5" : "config.json5";
```

Comment:
Good point, I added some extra tests.
Now tests will exit if NODE_ENV is not test
and if NODE_ENV is test, config.database.database will be set to "test"

I like 2 config files because it would be nice to allow the API to be on different ports so I can write tests while checking routes in browser, but since I made the changes above I can delete the test config file if you want

would just need to change 
```static port = 8080```
 in api.js to
 ```static port = config.api.port```
  and add a field for { api: { port: 8081 or 8080 } } in the config file.